auto_round/script/llm.py

```diff
@@ -104,6 +104,13 @@ def __init__(self, *args, **kwargs):
             help="minmax learning rate, if None, it will beset to be the same with lr",
         )
 
+        self.add_argument(
+            "--mem_per_param_scale",
+            default=13,
+            type=float,
```

Comment:
please make sure the command line usage is ok if you have not tested it . auto-round xxx --devices  0,1